script/input/1/phase-1b/template-farming-deploy.json

```
@@ -0,0 +1,3 @@
+{
+  "nst": "address"
```

Add missing `.admin` prop and some context regarding the config params.
```suggestion
  "nst": "address: the address for NST",
  "admin": "address: the owner of the contract (usually the MCD_PAUSE_PROXY address)"
```